frontend/scripts/gameController.js

```diff
@@ -14,6 +14,10 @@ const TETROMINO_SHAPES = {
 
 // Gets a random piece name from TETROMINO_SHAPES
 function getRandomPiece() {
+    rand = Math.floor(Math.random() * 7);
+    block_list = [TETROMINO_SHAPES.I_PIECE, TETROMINO_SHAPES.J_PIECE, TETROMINO_SHAPES.L_PIECE, TETROMINO_SHAPES.O_PIECE, TETROMINO_SHAPES.S_PIECE, TETROMINO_SHAPES.T_PIECE, TETROMINO_SHAPES.Z_PIECE]
+    shape = block_list[rand]
+    console.log(shape)
```

Comment:
Yep this works! 

If you wanted to add some slight improvements you could tie the random block generation to the length of TETROMINO.SHAPES

The function `Object.keys(TETROMINO_SHAPES)` would get you a list of the names of the shapes. This would mean that if you decided to remove a shape, or add a new shape, you wouldn't need to change this code

Reply:
Okay will do